### PR TITLE
Don't hard-code constants in scheduler_dag_execution_timing

### PR DESCRIPTION
**Pull request by ashb**
Slight "improvement" on #8949

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: scripts/perf/scheduler_dag_execution_timing.py [hunk 1]
```diff
@@ -131,11 +131,18 @@ def create_dag_runs(dag, num_runs, session):
     from airflow.utils import timezone
     from airflow.utils.state import State
 
+    try:
+        from airflow.utils.types import DagRunType
+        ID_PREFIX = f'{DagRunType.SCHEDULED}__'
```
Comment (kaxil) [inline scripts/perf/scheduler_dag_execution_timing.py]:
```suggestion
        ID_PREFIX = f'{DagRunType.SCHEDULED.value}__'
```

Comment (kaxil) [reply scripts/perf/scheduler_dag_execution_timing.py]:
```python console
In [1]: from airflow.utils.types import DagRunType

In [2]: DagRunType.SCHEDULED
Out[2]: <DagRunType.SCHEDULED: 'scheduled'>

In [3]: f'{DagRunType.SCHEDULED}__'
Out[3]: 'DagRunType.SCHEDULED__'

In [4]: f'{DagRunType.SCHEDULED.value}__'
Out[4]: 'scheduled__'
```

Comment (ashb) [reply scripts/perf/scheduler_dag_execution_timing.py]: Seems to work without it... curious.

Comment (ashb) [reply scripts/perf/scheduler_dag_execution_timing.py]: Oh, no it _doesnt_.

```python console
In [5]: f'{DRT.SCHEDULED}__'                                                                                                                                     
Out[5]: 'DagRunType.SCHEDULED__'
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**